### DISCUSSION TITLE
Unable to configure tailwind in Next.js project

### DISCUSSION THREAD

**Discussion by Pacheco95** [tailwindlabs/tailwindcss#6610] [Help] [answered]
Hello everyone. I'm trying to configure TW in my Next project alongside with typescript.
I have followed [these steps](https://tailwindcss.com/docs/guides/nextjs) but it seems to not work. The classes does not affect the css of the page.

Please help me!

Example repo https://github.com/Pacheco95/next-tailwind-issue-report

**Comment by aggreggator** [comment, marked as answer]
https://github.com/Pacheco95/next-tailwind-issue-report/blob/main/tailwind.config.js#L2
These paths are not present in your project.

**Reply by Pacheco95** [reply]
Woops. Sorry for my inattention :man_facepalming:

**Reply by aggreggator** [reply]
No worries. 
You've provided a repro repo and people are able to help.
